app/src/test/java/com/vero/tasky/auth/domain/usecase/ValidateNameUseCaseTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.vero.tasky.auth.domain.usecase
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class ValidateNameUseCaseTest {
+
+    private val validateNameUseCase = ValidateNameUseCase()
+
+    @Test fun `Validate name, too short name, result false`() {
```

Comment:
I personally have the @Test above the function. Not on the same line
I find it easier to read